### DISCUSSION TITLE
Create Folder

### DISCUSSION THREAD

**Discussion by j-taptok** [open-webui/open-webui#1182] [Q&A] [answered]
Hi,

Are there plans to add "Folders", I see tags for conversation but after heavily using your solution the need for folders is becoming critical. Conversations pile up and is hard to find them later on even with Tags.

**Comment by tjbck** [comment]
Hmm, I'm struggling to understand what folders could offer that tag feature can't, could you elaborate more on your workflow so we can better understand?

**Reply by j-taptok** [reply]
When you have too many conversations the list becomes extremely long. Tags don't hide the conversations.

This might be exclusive to my own organization approach, if this was the only request maybe is not worth the implementation.

**Reply by Lanhild** [reply]
I think the folder approach needs to be further studied. It's a need when you have many conversations, and folders are by definition easier to organize than many tags

**Reply by markwylde** [reply]
I think folders would be amazing and it's definitely something I wish open-webui had.

If you're really against folders, would there be any possibility to have a "group" view option in the chat, that could group chats.

For example, convert;

- Macbook Install brew (tag: hardware)
- Home made soup (tag: food)
- Refactor code (tag: code)
- Write a hello function (tag: code)

Into a collapsable grouped list like:

[-] Hardware:
- Macbook Install brew

[-] Food:
- Home made soup

[-] Code:
- Refactor code (tag: code)
- Write a hello function (tag: code)

[+] Songs:
[+] Company Name Ideas:

**Reply by tjbck** [reply]
I think I'll do a huge overhaul here in the near future, so definitely stay tuned!

**Comment by tjbck** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/0f5f52bd-a1b1-4ab1-874c-39afc4ce772f)

Implemented on dev!

**Reply by markwylde** [reply]
@tjbck you are a hero. This looks great. Thanks so much.
